Cargo.toml

```diff
@@ -42,6 +42,7 @@ isahc = { version = "0.9", optional = true, default-features = false, features =
 js-sys = { version = "0.3.25", optional = true }
 wasm-bindgen = { version = "0.2.48", optional = true }
 wasm-bindgen-futures = { version = "0.4.5", optional = true }
+futures-util = "0.3.1"
```

Comment:
I'd rather we not introduce `futures-util` into this crate; it comes with a lot of extra dependencies that have a negative impact on compilation times.

Reply:
If we have to, I'd rather we rely on `async-std` with `default-features = false` and `features = ["core"]` enabled. That will keep our dependency tree to a minimum in most cases.

Reply:
With some minor tweaks, I've removed futures-util.